tfe.go

```diff
@@ -509,6 +510,38 @@ func (c *Client) newRequest(method, path string, v interface{}) (*retryablehttp.
 	return req, nil
 }
 
+// Encode encodes the values into ``URL encoded'' form
+// ("bar=baz&foo=quux") sorted by key.
+func EncodeIncludeParams(v url.Values) string {
+	if v == nil {
+		return ""
+	}
+	var buf strings.Builder
+	keys := make([]string, 0, len(v))
+	for k := range v {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	for _, k := range keys {
+		vs := v[k]
+		if len(vs) > 1 {
+			val := strings.Join(vs, ",")
```

Comment:
I think there might be an issue here. By my reading, this changes the default behavior of url Encode to encoding multiple values as repeated keys (include=A&include=B) to _add_ a third key, (include=A&include=B&include=A,B)

I don't think this is what we want for include parameters, and probably not what we want universally, so maybe you should actually see if k == "include" before encoding as comma separated value. Comma-separated values aren't parsed by rails as an array automatically so it feels a little aggressive to always do this, even though I'm unaware of any other array-style get parameters in go-tfe

Reply:
geez :face_palm: , I left behind an import line between 528 and 529: `vs = [:0]`
This was working nonetheless because the right concatenated string (include=A,B) was being appended to the end of the slice, and rails is looking for a param named "include" and matching only that one occurrence. 
I will adjust.